lib/wifi_user/use_case/sponsor_users.rb

```diff
@@ -12,8 +12,8 @@ def execute(unsanitised_sponsees, sponsor)
 
     if whitelist_checker.execute(sponsor_address)[:success]
       sponsees = sanitise_sponsees(unsanitised_sponsees)
-      invite_sponsees(sponsor, sponsor_address, sponsees)
-      send_confirmation_email(sponsor_address, sponsees)
+      failed_sponsees = invite_sponsees(sponsor, sponsor_address, sponsees)
```

Comment:
I'm surprised by an `invite_sponsees` method returning a naked `failed_sponsees` - but the naming does make it clear what happens from here on out. Hmm.

Reply:
yeah, maybe this should return an object instead.
```
def invite_sponsees(sponsor, sponsor_address, sponsees) -> Hash[:succeeded, :failed]
```